lymproblems/timezonechange.py

```diff
@@ -0,0 +1,23 @@
+# Change the system's timezone to the same of SFO, USA
+
+import os
+import sys
+from .utils import system, success, fail
+
+
+def setup():
+    "Setup timezonechange"
+    pass  # Nothing to do.
+
+
+def verify():
+    "Verify timezonechange"
+
+    SFOtimezone = "/usr/share/zoneinfo/US/Pacific"
+    localtimezone = "/etc/localtime"
+
+    if os.popen("zdump {}".format(localtimezone)).read().strip(
```

Comment:
Here is the function to use <https://docs.python.org/2/library/os.path.html#os.path.realpath>

Reply:
@kushaldas I added the above function.